### PR TITLE
Cleanup to prepare for flutter web

### PR DESCRIPTION
**Pull request by jacob314**

### CODE REVIEW

File: packages/devtools_app/pubspec.yaml [hunk 1]
```diff
@@ -59,6 +59,27 @@ dev_dependencies:
   mockito: ^4.0.0
   test: ^1.0.0
   webkit_inspection_protocol: ^0.5.0
-  devtools: ^0.1.7
+  devtools: #^0.1.7
+    path: ../devtools
```
Comment (kenzieschmoll) [inline packages/devtools_app/pubspec.yaml]: Is this intentional?

File: packages/devtools_app/lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,101 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/main.dart]: Is this file a straight copy paste?

File: packages/devtools_app/pubspec.yaml (hunk 1 above)
Comment (jacob314) [reply packages/devtools_app/pubspec.yaml]: yes.

File: packages/devtools_app/lib/main.dart (hunk 2 above)
Comment (jacob314) [reply packages/devtools_app/lib/main.dart]: yep straight copy and paste from the now stripped down file under web/main.dart

**Pull request merged**